example/scalalib/web/11-todo-http4s-scalasql/mill

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env sh
```

Comment:
Shouldn't need the `mill` file in the examples, it gets filled in automatically when publishing

Reply:
Addressed [here](https://github.com/com-lihaoyi/mill/pull/4924/commits/ac52b64be0f2fdb0907ae37e90cb26701704387c)